data/obs-packaging/cc-oci-runtime/debian.rules

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/make -f
+include /usr/share/dpkg/default.mk
+export LANG=C
+%:
+	dh $@
+
+override_dh_auto_configure:
+	sh ./autogen.sh --disable-static \
```

Comment:
that's a good idea @jodh-intel for the time being I will update the options and raise an issue related to  qemu-lite build options.